workflow_capability_core/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```

Comment:
this should not be part of the archive.
(gitignore ?)

Reply:
I put in the ignore file

Reply:
The file still is in the pull request however...... I guess Patrick's remark was about removing this preferences file from the commit, right?

Reply:
Still not removed ?